### PR TITLE
Feature/broker listener

### PR DESCRIPTION
**Pull request by caleb-acosta**
To do:
 
- [ ] Unit Tests

### CODE REVIEW

**Comment by caleb-acosta** [comment]
I think the unit tests will be in a separated MR

**Comment by hochata** [comment]
> I think the unit tests will be in a separated MR

Sure, no hurries


Still, we need to setup a minimal coverage check in the CI...

File: Analyzer/Program.cs [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
+
+// Create 10 threads to listen messages
+for (int i = 1; i <= 10; i++)
+{
+    Thread workerThread = new Thread(new ParameterizedThreadStart(Setup));
+    workerThread.Start(configuration);
+}
+
+Console.WriteLine("Press any key to exit...");
+Console.ReadKey();
```
Comment (hochata) [inline Analyzer/Program.cs:36]: Is this really needed? 

I hoped to have a worker pool supervisor always on a loop

**Review by hochata — COMMENTED** [review]
I still need to test this locally, but it looks good.

File: Analyzer/Program.cs [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
+var queueName =
+    configuration.GetValue<string>("RabbitMQ:QueueName")
+    ?? throw new InvalidDataException("Missing QueueName configuration");
+
+// Create 10 threads to listen messages
+for (int i = 1; i <= 10; i++)
+{
+    var factory = new ConnectionFactory() { HostName = hostName };
+    var connection = factory.CreateConnection();
+    var worker = new Worker(queueName, reporter, connection);
```
[the remark this answers is not in the document]
Comment (caleb-acosta) [reply Analyzer/Program.cs:64]: Yes, in the Rabbitmq docs says each consume should have its own connection

**Pull request merged**